scarlet/source.py

```diff
@@ -390,11 +390,11 @@ def __init__(self, frame, sky_coord, observation, symmetric=True, monotonic=True
         if self.symmetric:
             if self.frame.psfs is None:
                 shape = (41, 41)
-                psf = generate_psf_image(gaussian, shape, amplitude=1, sigma=.9)
+                psf = generate_psf_image(gaussian, shape, amplitude=1, sigma=.9, normalize=False)
```

Comment:
Why not normalising?

Reply:
This part of the code will hopefully be obsolete soon, since it doesn't really work as well as we had hoped, but the PSF is only used to weight the centroid. So the PSF should have a value of 1 for the center pixel, which is achieved in the next line so there didn't seem to be a need to perform an extra normalization in `generate_psf_image` that will be undone anyway.